### ISSUE TITLE
Impossible to download with javascript disabled

### ISSUE THREAD

**Issue by jonas2515** [elementary/website#1784]
It's impossible to download Elementary OS if you don't allow the execution of js-code from jsdelivr.net and stripe.com.
For accessibility purposes, I think there should be at least one direct-download link on the Website.

**Comment by btkostner** [comment]
I'm going to mark this as invalid, as we use JS not only for Stripe, but for fetching the correct download link from our CDN, and counting downloads. Disabling or blocking JavaScript from running on the modern web is a disaster, and the user should expect things to break, not only for our website, but for most sites.
